src/commands/setup.js

```diff
@@ -17,15 +17,20 @@ async function getUserInfoFromApi() {
       end_date: moment().format('YYYY-MM-DDTHH:mm:ss.SSS'),
     },
   });
-  const startDate = completedReviews.body
-    .map(review => moment(review.assigned_at)) // returns date of review
-    .map(date => date.valueOf()) // returns date in Unix Time (milliseconds from 1970)
-    .reduce((acc, val) => { // returns the smallest number
-      if (acc < val) {
-        return acc;
-      }
-      return val;
-    });
+  let startDate;
+  if (completedReviews) {
```

Comment:
`completedReviews` is the response from the API, so it will always be truthy I think. You should check for `completedReviews.body.length`. Otherwise it's good.